api/HTMLVideoElement.json

```diff
@@ -14,7 +14,7 @@
             "version_added": "12"
           },
           "firefox": {
-            "version_added": "4"
+            "version_added": "3.5"
```

Comment:
Indeed, `HTMLVideoElement` had to have been shipped together with `HTMLMediaElement` and `HTMLAudioElement`.